packages/client/src/canvasEventHandler.ts

```diff
@@ -24,13 +24,19 @@ export abstract class CanvasEventHandlerBase implements CanvasEventHandler {
     this.canvasWrapper = canvasWrapper;
   }
 
+  getWorldPos(ev: MouseEvent | WheelEvent): { x: number, y:number } {
+    const canvasPos = this.canvasWrapper.eventCanvasPositionGetter.get(ev);
+    return this.canvasWrapper.canvasPointToWorld(canvasPos.x, canvasPos.y);
+  }
+
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  handleDownEvent(x: number, y: number): void {
+  handleDownEvent(ev: MouseEvent): void {
     // do nothing
   }
 
-  handleMoveEvent(x: number, y: number): void {
-    const rectShape = this.canvasWrapper.getLastRectShapeContaining(x, y);
+  handleMoveEvent(ev: MouseEvent): void {
+    var worldPos = this.getWorldPos(ev);
```

Comment:
use 'const' instead of var.

Prefer to use eslint and prettier. I think it would have replaced 'var' with 'const' here